projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployerV2.java

```diff
@@ -346,34 +345,37 @@ private KubernetesService.CronJob getCronJob(
 
     var jobCommand = jobCommandProvider.getJobCommand(jobName);
 
+    checkDefaultDeploymentResources(jobDeployment);
     String cpuRequest =
         Optional.ofNullable(jobDeployment.getResources())
             .map(DataJobDeploymentResources::getCpuRequestCores)
             .filter(cpuRequestCores -> cpuRequestCores > 0)
-            .map(String::valueOf)
+            .map(cpuRequestM -> String.valueOf(cpuRequestM.intValue()))
+            .map(cpuRequestM -> cpuRequestM + "m")
             .orElse(defaultConfigurations.dataJobRequests().getCpu());
-
+    checkDefaultCpuRequest(jobDeployment.getResources(), cpuRequest);
```

Comment:
In my opinion, a better place for these assignments is in the

`DeploymentModelConverter.toActualJobDeployment(desiredDataJobDeployment, desiredDeploymentVersionSha, lastDeployedDate);` 

Since the purpose of this method is just to construct the job.